css/obBase.css

```diff
@@ -202,13 +202,41 @@ form {
   width: calc(100% - 230px);
 }
 
-#pageNav {
+#appBar {
   position: fixed;
-  top: 0;
+  width: 100%;
+  z-index: 14;
+  background-color: #327EB8;
+  padding: 10px 8px;
+  box-sizing: border-box;
   -webkit-app-region: drag;
-  z-index: 13;
 }
 
+#appBar .windowControls {
+  position: absolute;
+  left: 8px;
+  top: 8px;  
+}
+
```

Comment:
hmm... the buttons are clickable for me and the Mac ones do have a hover state. I didn't put a hover state on the Win one. I left that for @morebrownies to do at his discretion.

Anyhow, maybe for some reason Windows needs that `no-drag` class. I'll add it in.